### ISSUE TITLE
Drop shadow on genes card

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/genepanel.iobio#264]
This card seems to have a different drop shadow to the other cards. Is this by design?

**Comment by adityaekawade** [comment]
Yes, it has a different drop to show that it is an active card. 
I can remove the extra drop shadow:
![Screen Shot 2018-09-05 at 12.58.21 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a68ffc37453d30014ef4f0d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b772a6454f66b7191a379ed0c2e59415d4fbc21254d58eeb9cb74fce36f34985619e880640a20cf3443a9eed41fed04ba208a1d87b5ed824e94fa2137a5dad11b61435c52dcbd4f3fb29843c259f348e74c52a0133762e571326e0a.png)

Also, I can remove the blue underline and make all the cards look similar. We had the blue underline to indicate that the card is selected. Now, since we have the modal do we need this blue underline?
![Screen Shot 2018-09-05 at 12.58.55 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a68ffc37453d30014ef4f0d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b772a6454f66b7191a379ed0c2e59415a4bbc21254d58eeb9cb74fce36f34985619e880640a20cf3443a9eed41fed04ba208a1d87b5ed824e94fa2137a5dad11b61435c52dcbd4f3fb2984cc259f348e74c52a0123462ef72326e0a.png)

**Comment by adityaekawade** [comment]
I have removed the extra shadow from this card for tomorrow. We will sort the position of this card after the session.
